app/services/dataworks_mappers/sdr.rb

```diff
@@ -0,0 +1,471 @@
+# frozen_string_literal: true
+
+module DataworksMappers
+  # Map fields from SDR cocina records to Dataworks metadata
+  # rubocop:disable Metrics/ClassLength
+  class Sdr < Base
+    include ActiveSupport::NumberHelper
+
+    # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
+    def perform_map
+      {
+        creators:,
+        titles:,
+        publisher:,
+        publication_year:,
+        subjects:,
+        contributors:,
+        descriptions:,
+        dates:,
+        language:,
+        version:,
+        identifiers:,
+        related_identifiers:,
+        sizes:,
+        formats:,
+        rights_list:,
+        funding_references:,
+        related_items:,
+        url:,
+        access:,
+        provider: 'SDR'
+      }.compact_blank
+    end
+    # rubocop:enable Metrics/MethodLength, Metrics/AbcSize
+
+    private
+
+    # Not mapping alternative/subtitles at the moment
+    def titles(element = source.description.title)
```

Comment:
Why is this accept parameters?

Reply:
Because it sometimes is called on the entire record (the default) and sometimes is called on related works when iterating over them, but in both cases has the same behavior.